### DISCUSSION TITLE
Review comments containing inline back-tick-quoted code strings collapses contiguous whitespace

### DISCUSSION THREAD

**Discussion by kpg-ideon** [community/community#37372] [Pull Requests]
While reviewing a coworker's code, involving the processing of text strings, my comment needed to include something like: `"hello world"` where the words `"hello"` and `"world"` would be separated by many whitespace characters. It needed to be visually clear there were multiple contiguous whitespace characters too.

However, when entering the back-tick-quoted, inline string with 7 spaces, instead of displaying `"hello_______world"` (where the underscore is instead a whitespace character) making it very clear I'm discussing a text string with many contiguous whitespace characters, it instead collapses the whitespace characters, just like non-back-tick-quoted text, displaying it the same as `"hello world"` (single whitespace character) which made it more difficult to communicate my intention regarding string processing.

I can work around it using a code fence block:
```ruby
  x = "hello       world"
  ...
```

Or by using a replacement character like an underscore as I did above.

However, I shouldn't have to work around this. If my text is back-tick quoted, please respect embedded whitespaces.

**Comment by ossanna16** [comment]
Hi there @kpg-ideon and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
